modules/apps/transfer/types/msgs.go

```diff
@@ -71,16 +64,6 @@ func NewMsgTransfer(
 	}
 }
 
-// Type implements legacytx.LegacyMsg
-func (MsgTransfer) Type() string {
-	return TypeMsgTransfer
-}
-
-// Route implements legacytx.LegacyMsg
-func (MsgTransfer) Route() string {
-	return RouterKey
-}
```

Comment:
I guess these are super old, which used to be used for msg routing from baseapp to module? Or at least that's my assumption

Reply:
correct!